verifier/verifier.go

```diff
@@ -204,15 +205,28 @@ func (v *verifier) processSignature(ctx context.Context, sigBlob []byte, envelop
 		}
 
 		if len(capabilitiesToVerify) > 0 {
-			response, err := executePlugin(ctx, installedPlugin, trustPolicy, capabilitiesToVerify, outcome.EnvelopeContent, pluginConfig)
+			var response *proto.VerifySignatureResponse
+			response, err = executePlugin(ctx, installedPlugin, trustPolicy, capabilitiesToVerify, outcome.EnvelopeContent, pluginConfig)
 			if err != nil {
 				return err
```

Comment:
Looks like this err is returned before printing out any log? Similar to the returns in this function above.

Reply:
updated.